project/Build.scala

```diff
@@ -258,14 +258,15 @@ object ScaldingBuild extends Build {
 
   lazy val scaldingParquet = module("parquet").settings(
     libraryDependencies ++= Seq(
-      "com.twitter" % "parquet-cascading" % "1.4.0",
+      "com.twitter" % "parquet-cascading" % parquetVersion,
+      "com.twitter" % "parquet-avro" % parquetVersion,
```

Comment:
well maybe go the other direction then -- treat parquet as a common thing, and push this up into scalding-avro, and put parquet-thrift in scalding-thrift?